Hardcore.lua

```diff
@@ -149,6 +149,7 @@ local ALERT_STYLES = {
 		alertSound = nil
 	},
 }
+Hardcore_Alert_Frame:SetScale(0.7)
```

Comment:
Couldn't figure out how to do this in XML. (Where I would prefer the scaling live)